src/App.tsx

```diff
@@ -16,35 +18,74 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section content">
-      <div className="buttons">
-        <button type="button" className="button is-info is-light">
-          Sort alphabetically
-        </button>
+  const [sortField, setsortField] = useState('');
```

Comment:
The function `setsortField` should be `setSortField` to follow the naming convention of React hooks, which typically use camelCase starting with 'set' for state update functions.